documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -36,14 +36,24 @@ stringData:
   user: <user> <1>
   password: <password> <2>
   cacert: | <3>
-    <engine_ca_certificate>
+    <ca_certificate>
   thumbprint: <vcenter_fingerprint> <4>
 EOF
 ----
 <1> Specify the vCenter user or the {rhv-short} {manager} user.
 <2> Specify the user password.
-<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
+<3> {rhv-short} only: {manager} CA certificate unless it was replaced by a third-party certificate, in which case see the note below. The {manager} CA certificate can be retrieved at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
 <4> VMware only: Specify the vCenter SHA-1 fingerprint.
++
+[NOTE]
+====
+If the {manager} CA certificate was replaced by a third-party certificate, replace `cacert:<ca_certificate>` in the `Secret` manifest with the following:
+
+`cacert: <manager_apache_certificate>` +
+`createdForProviderType: ovirt` +
+`createdForResource: <resource>` +
+`createdForResourceType: providers`
```

Comment:
I posted https://github.com/kubev2v/forklift/pull/246 to drop the last two (`createdForResource` and `createdForResource`) as they seem unused now. @qiyuann could you please double check me here, you said you've added it to the automation tests, right?

@RichardHoch besides dropping the last two if they are indeed not needed, the value of `createdForProviderType` can be either `ovirt`, `vsphere` (@yaacov can you please keep me honest that it's 'vsphere' and not 'vmware'? we don't use this value on the backend yet) and `openstack`

Reply:
>  @qiyuann could you please double check me here, you said you've added it to the automation tests, right?
>
Only with the label `createdForProviderType: ovirt`, manager CA could be retrieved when the cacert field is filled with 3rd-party CA.  The url must be defined as well, otherwise it will report missing protocol schema error when creating the secret.

For ovirt, `insecureSkipVerify: false` could be added automatically if it's not specified when creating secret. But for vsphere and openstack, there is no default definition for it. 
Is it a must to also add `insecureSkipVerify` option for vsphere and openstack?

The labels should be under metadata, not stringData:
```
metadata:
  name: <secret>
  namespace: {namespace}
  labels:
    createdForProviderType: <provider_type>
```

Reply:
Q: what does the label do ? it's nice, but why do we need it ?

Reply:
@ahadas @qiyuann Yes I remember that we discussed that, but for some reason it exists only for ovirt,  it can be easily added to other types as well.
the only thing was, IIRC for VMware and OpenStack, the key name was different then  insecureSkipVerify` . @yaacov is it still the case?